tests/test_runner/test_hooks.py

```diff
@@ -1121,7 +1121,7 @@ def test_wandb_hook():
     
     hook.wandb.init.assert_called_with()
     
-    hook.wandb.run.dir = runner.work_dir
+    hook.wandb.run.dir.return_value = runner.work_dir
     hook.wandb.save.assert_called_with(
         glob_str=hook.wandb.run.dir + '/*',
         base_path=hook.wandb.run.dir,
```

Comment:
In this test, `hook.wandb` is not `wandb.run`.  `wandb.init()` is a code to make an `wandb.run` object, so after initialization of wandb, every `wandb` is alias of  `wandb.run` (more details are [here](https://docs.wandb.ai/ref/python/init)). Therefore, I should use `wandb.run`  to call `wandb.run.dir`, but upper line defines `wandb` is MagicMock. Instad of modification of whole lines in this project for wandb, I just use `return_value` to return the work_dir of runner. This is the same work with `shutil.copy2` and `wandb.save` in wandb logger script.